src/CustomMimetypeHelper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Http\Message\MultipartStream;
+
+/**
+ * Let you add your own mimetypes.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class CustomMimetypeHelper extends ApacheMimetypeHelper
+{
+    /**
+     * @var array
+     */
+    private $mimetypes = [];
+
+    /**
+     * @param array $mimetypes
+     */
+    public function __construct(array $mimetypes = [])
+    {
+        $this->mimetypes = $mimetypes;
+    }
+
+    /**
+     * @param string $extension
+     * @param string $mimetype
+     *
+     * @return $this
+     */
+    public function addMimetype($extension, $mimetype)
```

Comment:
i am not sure we need an adder for this. the constructor should be enough, its not like we discover mime types at runtime - or is there a use case for that?